modules/rubiconAnalyticsAdapter.js

```diff
@@ -318,6 +367,74 @@ function setRubiconAliases(aliasRegistry) {
   });
 }
 
+function getRpaCookie() {
+  let encodedCookie = storage.getCookie(COOKIE_NAME);
+  if (encodedCookie) {
+    try {
+      return JSON.parse(window.atob(encodedCookie));
+    } catch (e) {
+      utils.logError(`Rubicon Analytics: Unable to decode ${COOKIE_NAME} value: `, e);
+    }
+  }
+  return {};
+}
+
+function setRpaCookie(decodedCookie) {
+  try {
+    storage.setCookie(COOKIE_NAME, window.btoa(JSON.stringify(decodedCookie)));
```

Comment:
why using a `cookie` rather than `local storage`?

Reply:
updated to localStorage